components/rsptx/validation/schemas.py

```diff
@@ -182,6 +182,9 @@ class LastPageDataIncoming(BaseModelNone):
     last_page_url: str  # = Field(None, alias="lastPageUrl") is the manual way
     course_id: str = Field(alias="course")
     completion_flag: int
+    pageLoad: bool
```

Comment:
This is fine, as long as they are always included on the POST.  If they are missing then the caller will get a 422 invalid request.

Reply:
Yes, these should always be set. See the new lines 313-315 in user_highlights.js.